### ISSUE TITLE
35.2 RVN Error load opencl module: clBuildProgram

### ISSUE THREAD

**Issue by o2genum** [NebuTech/NBMiner#184]
```
[19:33:23] INFO - ----------------------------------------------
[19:33:23] INFO - |                                            |
[19:33:23] INFO - |                                            |
[19:33:23] INFO - |                                            |
[19:33:23] INFO - |                                            |
[19:33:23] INFO - |                                            |
[19:33:23] INFO - |                                            |
[19:33:23] INFO - |                                            |
[19:33:23] INFO - |         NBMiner - Crypto GPU Miner         |
[19:33:23] INFO - |                    35.2                    |
[19:33:23] INFO - |                                            |
[19:33:23] INFO - ----------------------------------------------
[19:33:23] INFO - ------------------- System -------------------
[19:33:23] INFO - OS:     Windows 10 Version 2009, 10.0.19042
[19:33:23] INFO - CPU:    Intel(R) Core(TM) i5-7600K CPU @ 3.80GHz
[19:33:23] INFO - RAM:    3530 MB / 8080 MB
[19:33:23] INFO - VMEM:   42850 MB / 58080 MB
[19:33:23] INFO - CU_DRV: 11.2, 460.89
[19:33:23] INFO - CL_DRV: 20.11.2
[19:33:23] INFO - ------------------- Config -------------------
[19:33:23] INFO - ALGO:   kawpow
[19:33:23] INFO - URL:    stratum+tcp://rvn-pool.beepool.org:9531
[19:33:23] INFO - USER:   RW7W1X392kJy2Hf3WwHVGaUcY56Hy6R2YN.default
[19:33:23] INFO - TEMP:   90C
[19:33:24] INFO - ------------------- Device -------------------
[19:33:24] INFO -  |ID|PCI|  CC|Memory|CU|
[19:33:24] INFO - *| 0|  2| 800| 8192M|32| ASUS Radeon RX 470 Series
[19:33:24] INFO - *| 1|  3|  75| 6144M|30| GeForce RTX 2060
[19:33:24] INFO - *| 2|  9| 900| 8176M|56| Radeon RX Vega
[19:33:24] INFO - *| 3| 10| 800| 4096M|16| Radeon RX 560 Series
[19:33:24] INFO - *| 4| 11|  61| 8192M|20| GeForce GTX 1080
[19:33:24] INFO - *| 5| 13| 800| 8192M|36| Radeon RX 580 Series
[19:33:24] INFO - ----------------------------------------------
[19:33:24] INFO - kawpow - Logging in to rvn-pool.beepool.org:9531 ...
[19:33:25] INFO - kawpow - Login succeeded.
[19:33:25] INFO - API:  0.0.0.0:22333
[19:33:25] INFO - API server started.
[19:33:25] INFO - Device 0 started, Free mem = 8192 MB.
[19:33:25] INFO - Device 1 started, Free mem = 5141 MB.
[19:33:25] INFO - kawpow - New job: rvn-pool.beepool.org:9531, ID: 37d269b8, HEIGHT: 1550596, DIFF: 2.147G
[19:33:25] INFO - mining.extranonce.subscribe succeeded.
[19:33:25] INFO - Device 2 started, Free mem = 8176 MB.
[19:33:25] INFO - Device 3 started, Free mem = 4096 MB.
[19:33:25] INFO - Device 5 started, Free mem = 8192 MB.
[19:33:25] INFO - Device 4 started, Free mem = 7213 MB.
[19:33:26] ERROR - Error load opencl module: clBuildProgram
[19:33:26] WARN - QMutex: destroying locked mutex
[19:33:26] WARN - QMutex: destroying locked mutex

```

**Comment by o2genum** [comment]
34.5 works well

**Comment by nebutech-admin** [comment]
fixed in 36.0
